serviceWrappers/linux/kLiveController

```diff
@@ -23,36 +23,55 @@
 ###
 source /etc/rc.d/init.d/functions
 
+LOG_DIR=@LOG_DIR@
+PID_DIR=@RUN_DIR@
+USE_NVM=@USE_NVM@
+KLIVE_CONTROLLER_PREFIX="@KLIVE_CONTROLLER_PREFIX@"
+
+#default values
+if [ "${LOG_DIR}" = "@LOG_DIR@" ]; then
+    LOG_DIR=/var/log
+fi
+if [ "${PID_DIR}" = "@RUN_DIR@" ]; then
+    PID_DIR=/var/run
+fi
+if [ "${USE_NVM}" = "@USE_NVM@" ]; then
+    USE_NVM="1"
+fi
+if [ "${KLIVE_CONTROLLER_PREFIX}" = "@KLIVE_CONTROLLER_PREFIX@" ]; then
+    KLIVE_CONTROLLER_PREFIX=/opt/kaltura/liveController/latest
+fi
+
+
 NAME="kLiveController"
-KLIVE_CONTROLLER_PREFIX="/opt/kaltura/liveController/latest"
-NODE_PATH="$KLIVE_CONTROLLER_PREFIX/node_modules"
-FOREVER="$NODE_PATH/forever/bin/forever"
-APPLICATION_PATH="$KLIVE_CONTROLLER_PREFIX/lib/App.js"
-LOG_DIR="/var/log/liveController"
-PID_DIR="/var/run"
+NODE_PATH="${KLIVE_CONTROLLER_PREFIX}/node_modules"
```

Comment:
This is wrong because on OP ENVs, connection to the outside world is often not allowed and thus, NPMs cannot be retrieved from http://npmjs.com. We therefore package each such NPM module as an RPM and install it onto /usr/lib/npm_modules, and so, NODE_PATH should also be a token.